docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       ZOOKEEPER_TICK_TIME: 2000
 
   kafka:
-    image: confluentinc/cp-kafka:5.2.5-10
+    image: confluentinc/cp-kafka:7.3.0
```

Comment:
Is this version bump necessary for this new functionality to work?

Reply:
image: confluentinc/cp-kafka:5.2.5-10 does not have the default Kafka authorizer implementation class kafka.security.authorizer.AclAuthorizer. To address the ClassNotFoundException, I had to bump up the version. After bumping up the image version to 7.3.0, I executed the command bundle exec rspec  and ensured that there are no failures.